roles/openshift_logging/templates/elasticsearch.yml.j2

```diff
@@ -32,7 +32,7 @@ discovery:
 
 gateway:
   expected_master_nodes: ${NODE_QUORUM}
```

Comment:
Except that with those new parameters we'll still have to make changes to accommodate the new way to specify their values.  Seems better to apply this to the current code base, and then have that patch make the appropriate changes given the new parameters.